src/pokemon/types/ElectricPokemon.java

```diff
@@ -0,0 +1,137 @@
+package pokemon.types;
+
+import pokemon.Pokemon;
+
+public class ElectricPokemon extends Pokemon {
+    private static final int maxVoltage = 5000;
+    private int voltageLevel = 3590;
+    /**
+     * Constructs an ElectricPokemon with the given name, level, HP, food, and sound.
+     *
+     * @param name  the name of the ElectricPokemon
+     * @param level the level of the ElectricPokemon
+     * @param hp    the HP of the ElectricPokemon
+     * @param food  the food the ElectricPokemon likes
+     * @param sound the sound the ElectricPokemon makes
+     */
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound);
+        setType("Electric");
+        setFood("Pikachu Candy");
```

Comment:
Je zet hier voor alle Electric Pokemon dat ze Pickachu Candy eten. maar dit is natuurlijk alleen voor pikachu.

Reply:
Klopt.